lib/components/question_component.dart

```diff
@@ -0,0 +1,75 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+
+class QuestionComponent extends StatelessWidget {
```

Comment:
Name this as QuizQuestion to make sure people know this component is used in quiz